packages/@atjson/editor/src/mixins/editable-component.ts

```diff
@@ -0,0 +1,60 @@
+import WebComponent from './component';
+
+export default class EditableComponent extends WebComponent {
+
+  static style = `
+    :host {
+      position: relative;
+      outline: none;
+    }
+
+    @keyframes showControls {
```

Comment:
We could probably use `transition: opacity; opacity: 0;`  and `opacity: 1` for this.

Reply:
Oof, also realized that rendering all of these for a larger sized document will cause some slowness. It _should_ be fine for 95% of cases that we'll be working with for the moment, but it's a pretty hefty refactor of the code. Let's evaluate this to ensure that we're not gonna get ourselves stuck in a corner